can/interfaces/pcan/pcan.py

```diff
@@ -164,7 +163,7 @@ def __init__(self, channel='PCAN_USBBUS1', state=BusState.ACTIVE, bitrate=500000
         self.m_PcanHandle = globals()[channel]
 
         if state is BusState.ACTIVE or state is BusState.PASSIVE:
-            self.state = state
+            self._state = state
```

Comment:
Unless you're adding a property to expose this it should stay as `self.state` as it is part of the public Bus api.

https://python-can.readthedocs.io/en/master/bus.html#can.BusABC.state

Reply:
Hm yeah, I thought about that. But it declares `self._state` syntactically outside of `__init__`. Will revert it and add an ignore comment.